.rubocop.yml

```diff
@@ -1488,3 +1489,9 @@ Style/WhileUntilDo:
 Style/YodaCondition:
   Enabled: true
   EnforcedStyle: forbid_for_all_comparison_operators
+
+Style/DisableSyntax:
+  DisableSyntax:
+    - unless
```

Comment:
YES

Reply:
I don't actually mind `unless`, but I don't oppose this change either.

I was warned at a previous job that "people *will* hurt you" when I wrote an `unless... else` though. 😉 (`unless !something` is also cruel.)

Reply:
(The size of this PR seems to argue that it's well-entrenched and should maybe stay too?)